### PR TITLE
ScrollPosition.jumpTo call notifyListeners twice

### PR DESCRIPTION
**Pull request by nujz**
## Description

ScrollPosition.jumpTo call notifyListeners twice

## Tests

Add `ScrollPosition jumpTo()` test case in `scroll_position_test`.

### CODE REVIEW

**Review by Piinks — COMMENTED** [review]
Hi @nujz welcome :tada: and thank you for the contribution!
In order to land this change you will need to sign the cla, mentioned above by the google bot. 
This will also need to tests to verify the expected behavior - and prevent it from regressing in the future! :)

**Comment by nujz** [comment]
@googlebot I signed it!

File: packages/flutter/lib/src/widgets/scroll_position_with_single_context.dart [hunk 1]
```diff
@@ -197,7 +197,7 @@ class ScrollPositionWithSingleContext extends ScrollPosition implements ScrollAc
     goIdle();
     if (pixels != value) {
       final double oldPixels = pixels;
-      forcePixels(value);
+      correctPixels(value);
       notifyListeners();
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/scroll_position_with_single_context.dart]: Current:
```dart
      correctPixels(value);
      notifyListeners();
```

Suggest:
```suggestion
      forcePixels(value)
```

After reading the documentation for `correctPixels` and `forcePixels`, I think this should be the other way around, removing the call to `notifyListeners` in `jumpTo` and keeping the `forcePixels` call.

**Review by Piinks — COMMENTED** [review]
Hey @nujz thanks for adding a test! 

Is there an open issue that is referencing this bug?

**Comment by nujz** [comment]
@Piinks thanks.
I don't see related issues.

**Comment by Ahmadre** [comment]
I can confirm I got the same problem. I'll post the behavior as a gif.

**Comment by Piinks** [comment]
> I can confirm I got the same problem.

Thanks for the confirmation @Ahmadre !

**Review by Piinks — APPROVED** [review]
Flutter_LGTM
Thank you for the contribution! 🎉

**Pull request merged**